### PR TITLE
Writeup

### PR DESCRIPTION
**Pull request by nikkimoteva**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,128 +1,124 @@
...
+- [x] Table visualization for the submitted data
+- [x] View previously uploaded data (depending on accounts, can be per browser session or per user)
+- [x] Job tracking visualization/progress
 
-- Header menu
+#### 2-3 stretch requirements
+- [x] Multiple OAuth providers
+- [ ] Take as input other types of data
+- [x] Integrate other storage solutions into app (e.g. S3, kaggle, etc)
+- [ ] Job notification
```
Comment (KianShah) [inline README.md:55]: Users are now notified by email. Consider changing?

Comment (KianShah) [inline README.md]: Users are now notified by email. Consider changing?

File: README.md [hunk 2]
```diff
@@ -1,128 +1,124 @@
...
+- Tutorial page to demo the use of the application
+- Fancy data visualization
+- Other types of data such as JSON and XML, as well as ML vision models(images), NLP (Text), Time series data.
 
-**2-3 stretch requirements (plan to complete at least 1!)**
+<hr/>
 
-- Multiple OAuth providers
+### Technologies used in this project
+- **HTML/CS+JS:** In our project, we used JSX, JS front, and CSS for a lot of the stylings. We also used Material UI for styling and making our project more user friendly.
```
Comment (KianShah) [inline README.md]: CSS, not CS

File: README.md [hunk 3]
```diff
@@ -1,128 +1,124 @@
...
 
-- Job notification
+<hr/>
 
-- Integrate other storage solutions into app (e.g. S3, kaggle, etc)
+### Next Steps
+- **Job notification:** Due to time restriction, we were unable to work on this stretch goal, which is sending the user job notifications when their jobs have finished. However, currently the user is able to see the progress of their jobs on a progress bar we implemented on our jobs page. The job notification would ease the use of the application.
```
Comment (KianShah) [inline README.md]: Update this as well?

**Review by KianShah — CHANGES_REQUESTED** [review]

**Review by robertmzm — APPROVED** [review]
Looks all good

**Review by KianShah — APPROVED** [review]

**Pull request merged**